migrations/20201221143520-create-blog.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, DataType) => {
+    await queryInterface.createTable('Blogs', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataType.INTEGER
+      },
+      title: {
+        type: DataType.STRING
+      },
+      description: {
+        type: DataType.TEXT
+      },
+      userId: {
+        type: DataType.INTEGER
+      },
+      status: {
+        type: DataType.INTEGER
+      },
+      deletedAt: {
+        type: DataType.DATE
+      },
+      createdAt: {
```

Comment:
Update CreateAt and UpdateAt such that they do not need to be manually updated. should always have default values, unique property primary keys and foreign keys define where it would be not null with each column as needed.